src/main/scala/beam/sim/BeamHelper.scala

```diff
@@ -377,12 +376,18 @@ trait BeamHelper extends LazyLogging with BeamValidationHelper {
   }
 
   def vehicleEnergy(beamConfig: BeamConfig, vehicleTypes: Map[Id[BeamVehicleType], BeamVehicleType]): VehicleEnergy = {
-    val baseFilePath = Paths.get(beamConfig.beam.agentsim.agents.vehicles.vehicleTypesFilePath).getParent
+    var vehiclePaths = IndexedSeq(
+      Paths.get(beamConfig.beam.agentsim.agents.vehicles.vehicleTypesFilePath).getParent.toString
+    )
+    beamConfig.beam.agentsim.agents.freight.vehicleTypesFilePath
+      .map(freightVehicleTypesFilePath => Paths.get(freightVehicleTypesFilePath).getParent.toString)
+      .foreach(freightVehiclePath => vehiclePaths = vehiclePaths :+ freightVehiclePath)
+
```

Comment:
```suggestion
    val vehiclePaths =
      Paths.get(beamConfig.beam.agentsim.agents.vehicles.vehicleTypesFilePath).getParent.toString +:
      beamConfig.beam.agentsim.agents.freight.vehicleTypesFilePath
        .map(freightVehicleTypesFilePath => Paths.get(freightVehicleTypesFilePath).getParent.toString)
        .toIndexedSeq
```
I personally find hard to understand when an external collection gets changed in a loop. Hence, I would suggest this change.